spectral_cube/tests/test_projection.py

```diff
@@ -138,6 +138,9 @@ def test_self_arith(LDO, data):
 
     image = data
     p = LDO(image, copy=False)
+    p._wcs = WCS()
```

Comment:
Is this to hack around not passing a WCS when the LDO is created?

Can we pass this directly to the LDO creation (as the intended use case)?

Reply:
One more question: should we allow an LDO to be created _without_ a WCS? Or force raising an error?

Reply:
resolved this by creating w/WCS and not allowing creation w/o WCS